packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/DestinationNetworkBox.tsx

```diff
@@ -121,13 +121,21 @@ export function DestinationNetworkBox({
                       prefix={selectedToken ? '' : 'Balance: '}
                     />
                     {!isDepositMode && (
-                      <ETHBalance balance={ethParentBalance} />
+                      <ETHBalance
+                        balance={ethParentBalance}
+                        on={NetworkType.childChain}
```

Comment:
shouldn't that be `NetworkType.parentChain`?

Reply:
Updated - good catch